src/storage/query/QueryBaseProcessor.inl

```diff
@@ -170,6 +171,53 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts(const REQ&
     return cpp2::ErrorCode::SUCCEEDED;
 }
 
+template<typename REQ, typename RESP>
+bool QueryBaseProcessor<REQ, RESP>::checkDataExpiredForTTL(RowReader* reader) {
+    auto schema = reader->getSchema();
```

Comment:
It is not correct.  You should use schemaMan_ directly.

Reply:
Good Catch！